infrastructure/kubernetes/util/challenge-skeleton/config/network.yaml

```diff
@@ -4,8 +4,6 @@ metadata:
   name: "challenge-skeleton-lb-service"
 spec:
   type: "LoadBalancer"
-  # needs to be a premium IP
```

Comment:
IMO having a way to define a static IP is important and not everyone wants to setup dns :). Should we improve the comment instead of removing it or do you have a better workflow in mind?

Reply:
DNS, mostly. If there's a way to keep the DNS name in sync with the IP, it's probably fine to use a dynamic IP?

Reply:
Yeah, makes sense as long as the cloud dns is not too painful to set up. :)

Reply:
It's ok, just an NS record. Main downside is that propagation takes time, but with low TTL, it should be OK. And it's cheaper to have a DNS zone than a IP address I think.